nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -14,6 +14,39 @@ let
       exec "$@"
     '';
 
+  # https://wiki.archlinux.org/index.php/Talk:Bluetooth_headset#GDMs_pulseaudio_instance_captures_bluetooth_headset
+  pulseConfig = pkgs.writeText "default.pa" ''
+    #!/usr/bin/pulseaudio -nF
```

Comment:
This path won't exist on nixos.

Reply:
Does this need an executable bit? ( `writeText` doesn't do that)

Reply:
Ouch, nope, not needed at all (only useful if you want to apply configuration manually). As you can imagine this was a quick copy-paste from wiki.